pipelines/managed/fbc-release/fbc-release.yaml

```diff
@@ -580,6 +580,113 @@ spec:
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      runAfter:
+        - collect-data
+    - name: collect-index-images
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: $(params.taskGitUrl)
+          - name: revision
+            value: $(params.taskGitRevision)
+          - name: pathInRepo
+            value: tasks/managed/collect-index-images/collect-index-images.yaml
+      params:
+        - name: buildTimestamp
+          value: $(tasks.add-fbc-contribution-to-index-image.results.buildTimestamp)
+        - name: internalRequestResultsFile
+          value: $(tasks.add-fbc-contribution-to-index-image.results.internalRequestResultsFile)
+        - name: ociStorage
+          value: $(params.ociStorage)
+        - name: sourceDataArtifact
+          value: "$(tasks.add-fbc-contribution-to-index-image.results.sourceDataArtifact)"
+        - name: dataDir
+          value: $(params.dataDir)
+        - name: trustedArtifactsDebug
+          value: "$(params.trustedArtifactsDebug)"
+        - name: taskGitUrl
+          value: "$(params.taskGitUrl)"
+        - name: taskGitRevision
+          value: "$(params.taskGitRevision)"
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      runAfter:
+        - add-fbc-contribution-to-index-image
+    - name: create-pyxis-image
+      retries: 5
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: $(params.taskGitUrl)
+          - name: revision
+            value: $(params.taskGitRevision)
+          - name: pathInRepo
+            value: tasks/managed/create-pyxis-image/create-pyxis-image.yaml
+      params:
+        - name: server
+          value: $(tasks.collect-pyxis-params.results.server)
+        - name: pyxisSecret
+          value: $(tasks.collect-pyxis-params.results.secret)
+        - name: rhPush
+          value: "true"
+        - name: snapshotPath
+          value: "$(tasks.collect-index-images.results.indexImageSnapshot)"
+        - name: dataPath
+          value: "$(tasks.collect-data.results.data)"
+        - name: ociStorage
+          value: $(params.ociStorage)
+        - name: sourceDataArtifact
+          value: "$(tasks.collect-index-images.results.sourceDataArtifact)"
+        - name: dataDir
+          value: $(params.dataDir)
+        - name: trustedArtifactsDebug
+          value: "$(params.trustedArtifactsDebug)"
+        - name: taskGitUrl
+          value: "$(params.taskGitUrl)"
+        - name: taskGitRevision
+          value: "$(params.taskGitRevision)"
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      when:
+        - input: $(tasks.add-fbc-contribution-to-index-image.results.mustPublishIndexImage)
+          operator: in
+          values: ["true"]
+      runAfter:
+        - collect-pyxis-params
+        - collect-index-images
+        - publish-index-image
```

Comment:
The `create-pyxis-image` task need to be ran in all the cases. It can be placed behand `pushlish-index-image` . I need to be after `add-fbc-contribution-to-index-image` or earlier.

Reply:
> It can not be placed behand pushlish-index-image

Why not? My understanding was that we only want to create the image in Pyxis if it's gonna be published. Otherwise it's a throwaway image and we don't care. But I'll let @emilyzheng correct me if I'm wrong.

Reply:
Martin is correct. It has to be after publish.

Reply:
Yes, we don't create pyxis image (make it visible in catalog page) if it's not published.

Reply:
I was asked why FBC happy path passed, but hotfix not.

Reply:
@mmalina `create-pyxis-image` has the same `when` as `publish-index-image`. Did I miss anything?

Reply:
Hmm, so now I wonder why. I know you said it would not be used in happy path, but as I said above, I think the task will be triggered even if the publish task is skipped. Of course I might be wrong, but I did test this in an experiment like a month ago. Would be good to see the actual fbc happy path run to see what happened there.

Reply:
Ah, good. I should have checked. I assumed we relied on the skipping. So all good now.

> I was asked why FBC happy path passed, but hotfix not.

Yeah, ok.